deploy/build-server/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM ubuntu:18.04
+
+USER root
```

Comment:
It would be really good to see comments on everything in here.  So, why is this container running as `root`?